pages/tools.jsx

```diff
@@ -0,0 +1,151 @@
+          <a href={this.props.activityLink}>{this.props.activityTitle}</a>
+        </div>;
+    }
+    return (
+      <div className="col-sm-4 col-md-4 col-lg-4 tools-col">
+        <Illustration
+          link={this.props.link}
+          height={150} width={200}
+          src1x={this.props.src1x}
+          src2x={this.props.src2x}
+          alt=""
+          verticalLayout>
+          <h2><a href={this.props.link}>{this.props.name}</a></h2>
+          <p>{this.props.desctiption}</p>
+        </Illustration>
+        {sampleActivity}
+      </div>
+    );
+  }
+});
+
+var ToolsSection = React.createClass({
+  tools: [
+    {
+      name: "X-Ray Goggles",
+      desctiption: "This code inspector lets you view and remix the code of your favorite web pages.",
+      link: "https://webmaker.org/goggles",
+      src1x: "/img/pages/tools/xray-goggles.png",
+      src2x: "/img/pages/tools/xray-goggles@2x.png",
+      activityTitle: "Hack the News",
+      activityLink: "http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session02-hackthenews.html"
+    },
+    {
+      name: "Thimble",
+      desctiption: "This code editor helps you learn HTML and CSS by creating and remixing Web projects.",
+      link: "https://thimble.webmaker.org",
+      src1x: "/img/pages/tools/thimble.png",
+      src2x: "/img/pages/tools/thimble@2x.png",
+      activityTitle: "Image Seeking for Fantastic Visual Metaphors",
+      activityLink: "https://cogdog.makes.org/thimble/LTEyMjQ4NjUyOA==/imageseeking-for-fantastic-visual-metaphors"
+    },
+    {
+      name: "Webmaker",
+      desctiption: "Webmaker lets you create, discover and share content in your language on your mobile device.",
+      link: "https://webmaker.org",
+      src1x: "/img/pages/tools/webmaker.jpg",
+      src2x: "/img/pages/tools/webmaker@2x.jpg"
+    }
+  ],
+  render: function() {
+    return (
+      <section>
+        <div className="row">
+          { this.tools.map(function(tool, i) {
+              return <ToolsColumn {...tool} key={i} />;
+            })
+          }
+        </div>
+      </section>
+    )
+  }
+});
+
+var ToolsPage = React.createClass({
+  statics: {
+    pageTitle: 'Tools',
+    pageClassName: 'tools-page'
+  },
+  render: function(){
+    return (
+      <div className="inner-container">
+        <ToolsIntro/>
+        <ToolsSection/>
+        <section>
+          <div className="row">
+            <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+              <p className="callout-heading">Visit our <Link to="activities">Teaching Activities page</Link> for more examples of how to use these tools in your practice.</p>
+            </div>
+          </div>
+        </section>
+        <img src="/img/pages/tools/svg/icon-horizontal-divider.svg" width={292} alt="" className="center-block horizontal-divider"/>
+        <section>
```

Comment:
Hmm, I guess I'm curious if it's just coincidentally the same--if we're likely to change it to diverge from Teach Like Mozilla's footer CTA section anytime soon?  If that's the case, then it might be a good idea to just leave it as is, rather than refactoring things now and then un-refactoring when we need to change things again.

If we're unsure, I think it's probably OK to just leave this as-is for now, since this is just content and not functionality.